src/databases/PLY/avtPLYWriter.C

```diff
@@ -224,35 +224,34 @@ avtPLYWriter::GetCombineMode(const std::string &) const
 // Modifications:
 //   Justin Privitera, Mon Aug 21 15:54:50 PDT 2023
 //   Changed ColorTableAttributes `names` to `colorTableNames`.
+// 
+//   Justin Privitera, Fri Nov  3 15:25:32 PDT 2023
+//   Uses color table utility methods to improve performance.
```

Comment:
The main loop here seems almost the same. So, I am trying to understand what performance issue was improved (revision comment)